README.md

```diff
@@ -1,18 +1,15 @@
 # Project Name
 
-to do list
-
-# Description
-
-the second project in the second week of the second module.
-a simple but useful to do list
+set up the webpack
 
 ## Built With
 
 - Major languages
 - Frameworks
 - Technologies used
```

Comment:
Please mention the technologies which are being used to build this project 👍🏻

Reply:
hello @zainsadaqat  can you please watch this short video 
i explained everything in it

https://www.loom.com/share/2bba079a20164f4d9d7b17494c58fd20